libs/platform/table/components/table-view-settings-dialog/filtering/filters-list-step.component.html

```diff
@@ -7,7 +7,12 @@ <h4 fd-title [headerSize]="4">{{ 'platformTable.filterDialogFilterTitle' | fdTra
         <li fd-list-group-header>{{ 'platformTable.filterDialogFilterBy' | fdTranslate }}</li>
 
         @for (filter of filters; track filter) {
-            <li fd-list-item [interactive]="true" (click)="selectFilter.emit(filter)">
+            <li
+                fd-list-item
+                [interactive]="true"
+                class="fd-list__item_cursor-pointer"
```

Comment:
this should be fixed in fund-styles rather than adding a new class. If you want a temp fix better use dynamic style to add the cursor, something like:
[style.cursor]="interactive ? 'pointer' : 'auto'"

Reply:
I created a fix for this in Fd styles: https://github.com/SAP/fundamental-styles/pull/5678